app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/what-to-expect-in-virtual-coffee.mdx

```diff
@@ -0,0 +1,277 @@
+  	Coffee Table Groups
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/lunch-and-learns">
+  	Lunch & Learns
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/lightning-talk">
+  	Lightning Talk
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/monthly-challenges">
+  	Monthly Challenges
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/hacktoberfest-initiative">
+  	Hacktoberfest Initiative
+  </Link>
+
+Every Monday, we drop the week's list of events in the `#announcement` channel on Slack. We also drop the schedule on the day of the events. There will be another announcement to join the events in the `#vc-events` channel 10 minutes before the events start, along with the link to join.
+
+You can find the complete schedule of the events on the <Link to="/events">Events</Link> page.
+
+### Joining Events
+
+- Head over to the `#vc-events` channel on Slack.
+- Click the "Join Event" button to join the event.
+
+Now you will be directed to the event's Zoom.
+
+</TextContainer>
+
+<TextContainer>
+
+## Virtual Coffee on Slack
+
+<LeadText>
+
+Much of Virtual Coffee occurs on Slack. We have channels in our Slack to accommodate our members, and we encourage you to participate in Slack to get the most out of Virtual Coffee!
+
+</LeadText>
+
+### Slack Channels
+
+See our <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/slack-channels-guide">Virtual Coffee Slack Channels Guide</Link> for the complete list of our Slack channels.
+
+And read our tips around Slack in the next section if you need to familiarize yourself with Slack.
+
+### Tips Around Slack
+
+#### ➡️ Using Tags
+
+You can tag members (e.g., `@name`) to notify them about your thread and draw their attention.
+
+But unless you are a Maintainer or granted permission by a Maintainer, don't use `@here` or `@channel`.
+Using these tags will notify everyone in the channel. It might be an afternoon at your time, but it is midnight in some parts of the world. And we want to respect all of our members.
+
+#### ➡️ One Topic, One Thread
+
+Keep one topic in one thread. If you need to add more information to your message or you want to reply to a message, do that _in the thread_ and not create a new message to avoid confusion. Doing this will keep the topics and the conversations in the channels easy to follow by other members.
+
+##### On a Computer
+
+- Hover over the thread.
+- Click "Reply to thread".
+
+##### On a Phone
+
+- Press on the thread.
+- Click "Reply to thread".
+
+#### ➡️ Bookmarks and Pinned Messages
+
+You can check bookmarks and pinned messages on the top of each channel to find important and helpful links or resources around the community.
+
+#### ➡️ Including Code Snippets in a Message
+
+When you need to share code snippets in your message, you want to ensure it's well-formatted to make it readable and easy to understand.
```

Comment:
There are code and code block buttons in the text formatting menu. You can also create this formatting with backticks.

Reply:
Is this to replace the line 163, or adding to the paragraph?

Reply:
adding to the paragraph